adal/src/main/java/com/microsoft/aad/adal/AcquireTokenRequest.java

```diff
@@ -737,6 +759,19 @@ private void waitingRequestOnError(final CallbackHandler handler, final Authenti
         }
     }
 
+    private String checkPermission(final String permissionName) {
```

Comment:
We do have this checkPermission() function in BrokerProxy. It is a private method belong to BrokerProxy class. However,I have a concern to change it to package visible. 
AcquireTokenRequest.mBrokerProxy is the object of Interface IBrokerProxy. If we change checkPermission to package visiable, to call this function, we could do
    BrokerProxy object = (BrokerProxy) mBrokerProxy;
and call brokerproxy.checkPermission().